Team_Roster/migrations/alembic.ini

```diff
@@ -0,0 +1,50 @@
+# A generic, single database configuration.
```

Comment:
why is the migration being done through SQAlchemy? Can you please add a create table script to our UpdateDatabase Script so that it is created with everything else?